### PR TITLE
feat(suite): Add shortcut for settings

### PR DESCRIPTION
**Pull request by jvaclavik**
## Related Issue

Resolve

### CODE REVIEW

File: packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx [hunk 1]
```diff
@@ -29,5 +29,11 @@ export const useAppShortcuts = () => {
             );
             e.preventDefault();
         }
+
+        // press CMD + , to show Settings
+        if (metaKey && e.code === 'Comma' && selectedDevice) {
```
Comment (peter-sanderson) [inline packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx]: future idea: As keybinds are gonna grow, we may consider refactor those IFs into some map.

Comment (peter-sanderson) [reply packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx]: nit: `selectedDevice !== undefined`?

**Review by peter-sanderson — APPROVED** [review]

File: packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx (hunk 1 above)
Comment (jvaclavik) [reply packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx]: > future idea: As keybinds are gonna grow, we may consider refactor those IFs into some map.

Yes, totally. We are going to improve shortcuts across the app, so this should be part of it.

File: packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx [hunk 2]
```diff
@@ -29,5 +30,11 @@ export const useAppShortcuts = () => {
             );
             e.preventDefault();
         }
+
+        // press CMD + , to show Settings
+        if (metaKey && e.code === 'Comma' && isDeviceSelected) {
```
Comment (tomasklim) [inline packages/suite/src/components/suite/layouts/SuiteLayout/useAppShortcuts.tsx]: we have file for storing "code" `packages/components/src/constants/keyboardEvents.ts`

**Pull request merged**